prototype/tests/test_views.py

```diff
@@ -0,0 +1,88 @@
+from unittest.mock import patch
+
+from django.urls import reverse
+
+# from bs4 import BeautifulSoup
+
+from core.tests.helpers import create_response
+
+
+@patch('directory_cms_client.client.cms_api_client.lookup_by_slug')
+def test_prototype_landing_page_news_section(mock_get_page, client, settings):
+    settings.FEATURE_FLAGS['PROTOTYPE_PAGES_ENABLED'] = True
+    settings.FEATURE_FLAGS['NEWS_SECTION_ENABLED'] = True
+
+    url = reverse('prototype-landing-page')
+
+    page = {
+        'news_title': 'News',
+        'news_description': '<p>Lorem ipsum</p>',
+        'articles': [
+            {'article_title': 'News article 1'},
+            {'article_title': 'News article 2'},
+        ],
+    }
+
+    mock_get_page.return_value = create_response(
+        status_code=200,
+        json_body=page
+    )
+    response = client.get(url)
+
+    assert response.status_code == 200
+    assert response.template_name == ['prototype/landing_page.html']
+    print(response.context_data)
```

Comment:
I suspect this was erroneously committed

Reply:
yarp